test/extensions/filters/network/wasm/wasm_filter_test.cc

```diff
@@ -76,8 +75,14 @@ TEST_P(WasmNetworkFilterTest, BadCode) {
 
 // Test happy path.
 TEST_P(WasmNetworkFilterTest, HappyPath) {
-  setupConfig(TestEnvironment::readFileToStringForTest(TestEnvironment::substitute(
-      "{{ test_rundir }}/test/extensions/filters/network/wasm/test_data/logging_cpp.wasm")));
+  std::string code;
+  if (GetParam() != "null") {
+    code = TestEnvironment::readFileToStringForTest(TestEnvironment::substitute(
+        "{{ test_rundir }}/test/extensions/filters/network/wasm/test_data/test_cpp.wasm"));
+  } else {
+    code = "NetworkTestCpp";
+  }
```

Comment:
Nit: could you use consistent style between `test/extensions/filters/network/wasm/config_test.cc ` and `test/extensions/filters/network/wasm/wasm_filter_test.cc`? The same code is written differently in both places in this PR.

Reply:
Done